src/components/toolbar-menu/toolbarComponent.ts

```diff
@@ -73,6 +75,36 @@ export class ToolbarController {
   }
 
   /**
+   * Helper method for getting string value of {@link ToolbarType.BUTTON_SELECT}
+   * @memberof ToolbarController
+   * @function getToolbarListType
+   * @returns {string}
+   */
+  public getToolbarListType(): string {
+    return ToolbarType.BUTTON_SELECT.toString();
```

Comment:
I was thinking the just a typescript enum: https://www.typescriptlang.org/docs/handbook/enums.html